pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -1583,14 +1583,14 @@
             <item row="1" column="0">
              <widget class="QCheckBox" name="osdShowFPS">
               <property name="text">
-               <string>Show FPS</string>
+               <string>Show Internal FPS</string>
```

Comment:
It's both.

Reply:
This and "Show Game Frame Rate" are actually the same option, but for some reason, it had very different names between the option name and the tooltip name. I'd say this should be unified and whatever additional information you want to give to the player should be either on the first two places or on the tooltip contents.

Reply:
There is no "Show Game Frame Rate" option, I have no idea where you saw that. So if you could change this one back, as Internal FPS is not correct.

Reply:
Show FPS will both show the G: Game speed but internal is just what we call it as a nickname for coders which are minority, and V: Emulation Speed in how much it refreshes the application itself but games might still feel slow hence both FPS.

Reply:
@refractionpcsx2 
![Sin título](https://user-images.githubusercontent.com/5641681/221405207-948ab16f-3331-42af-91ab-29c733dff203.jpg)

Reply:
Ah the tooltip was incorrect, that explains it.